src/chrome/browser/ui/views/chrome_views_delegate_linux.cc

```diff
@@ -13,7 +13,7 @@
 #include "ui/base/resource/resource_bundle.h"
 #include "ui/views/linux_ui/linux_ui.h"
 
-#if defined(OS_WEBOS)
+#if defined(USE_CBE) // AGL:SINGLE_WINDOW_MODE
```

Comment:
CBE is not for this. It is for the single library for WAM and browser. I.e. we could build WAM only or browser only without CBE and we would still need this for webOS and AGL.
You should create a specific build define (i.e. USE_SINGLE_WINDOW_MODE) replacing these usages of OS_WEBOS, and enable it by default both for agl and webos.
I think best approach would be:

- Adding to src/neva/neva.gni declare_args second group a new arg "use_single_window_mode = is_webos || is_agl". The idea is that by default it is enabled both in webos and agl.
-  Add to src/neva/BUILD.gn a check for use_single_window_mode, so we define USE_SINGLE_WINDOW_MODE if use_single_window_mode GN var is declare.
- And yes, just replace the usages of OS_WEBOS with this new variable.

Reply:
@jdapena, thanks for the guide. I updated the branch. Please take a look again.